### ISSUE TITLE
Resizing and Orientation change does not update the position of the callout

### ISSUE THREAD

**Issue by dylankil** [OfficeDev/office-ui-fabric-react#3669]
### Bug Report

Package version: current
Browser and OS versions: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix?
Yes, but need to on-board to the fabric react project.

Requested priority: Normal

Products/sites affected: All

#### Describe the issue:

When a callout is open at its target element rotate the device (surface or mobile), the callout is no longer mapped to its target element

#### Actual behavior:

callout is no longer mapped to its target element

#### Expected behavior:

callout is mapped to its target element

**Comment by natalieethell** [comment]
Hey Dylan! The current behavior of the Callout is to dismiss on resize. In this case, rotating is the same idea, as the window resizes in that scenario. Thus, I think this issue would be a question of design. @betrue-final-final what do you think? Should we maintain the current behavior or maybe overhaul the dismiss logic?

**Comment by dylankil** [comment]
Hi, Natalie.

We’re using the Callout for the FRE of the Immersive Reader application, with this UX as a consideration it’s probably bad to dismiss on resize?

**Comment by natalieethell** [comment]
Hey Dylan, for clarity, what is the FRE of the Immersive Reader application? Sorry, I'm not too familiar with it. Would you be able to provide a screen capture? Thanks!

**Comment by dylankil** [comment]
Hello, Natalie.

Please find the attached.

If you’d like to check out the first run experience please visit https://onenote.com/learningtools

[cid:image001.png@01D3F81B.8E1850C0]

**Comment by betrue-final-final** [comment]
A screenshot would be great.

**Comment by natalieethell** [comment]
Thanks, Dylan! Here's a screen capture of the Callout disappearing on resize for visibility. 
![immersive reader resize](https://user-images.githubusercontent.com/14134000/40746917-911cf936-6410-11e8-8b69-15d3292c3364.gif)

**Comment by dylankil** [comment]
Yes, in this case it behaves as expected, unfortunately not so good of a first run user experience, would be nice to have the option not to dismiss on resize in some scenarios.

**Comment by betrue-final-final** [comment]
Fallouts like menus, are meant to be light dismiss objects, so maybe you should consider another surface for this if it needs to stay up. However I'm not sure that someone would invoke this callout, then resize the window before reading it. That's not how people do things unless you require someone to do it for some reason?

**Comment by dylankil** [comment]
On devices users may tilt to portrait or landscape. This is a common scenario we should probably support. I’d be happy to introduce you to the Learning Tools design lead for a better explanation. The answer may be to use a dialog with pivots on small screens, but this is what we are using at the moment until we make further changes.

We hacked-together our own solution for now, but it would be better to use a Fabric UI solution.


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by betrue-final-final** [comment]
That would be great. I'm interested in ensuring we have the best platform, so I would like to know their thinking.

**Comment by natalieethell** [comment]
Hey @dylankil just wanted to see if you had any updates on this? Thanks!
